appveyor.yml

```diff
@@ -0,0 +1,37 @@
+image: Visual Studio 2019
+
+environment:
+  host: x86_64-pc-windows-msvc        # Triple of host platform
+  matrix:
+    - platform: x86_64                # Name (is not used other than naming things)
+      target: x86_64-pc-windows-msvc  # Triple of target platform
+      channel: nightly                # Rust release channel (stable/beta/nightly/nightly-2018-12-01)
+
+install:
+    - git submodule update --init
+    - appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe  # Downloads Rustup-init
+    - rustup-init -y --default-toolchain %channel% --default-host %host%      # Installs Rust
+    - set PATH=%PATH%;%USERPROFILE%\.cargo\bin;%APPVEYOR_BUILD_FOLDER%        # Adds Rust tools (Cargo, Rustup, etc.) to path
+    - rustc -V                        # Prints Rust version
+    - cargo -V                        # Prints Cargo version
+    - rustup target add %target%      # Adds target platform to Rust
+
+build_script:
+    - cargo build --release --target=%target%      # Builds file defined in Cargo.toml (default main.rs)
+
+test_script:
+    - cargo test --target=%target% --verbose       # Runs tests in "src" and "tests" folders
+#    - cargo bench --target=%target% --verbose      # Runs benchmarks in "src/lib.rs"
```

Comment:
Can we remove this for now?

Reply:
Oh I meant removing just the bench part which was commented out anyway. The tests on Windows might be handy.

Reply:
Oh I meant to remove the bench one as it was commented out anyway. We can keep the tests as they might be useful to run on Windows as well.